lms/static/coffee/src/instructor_dashboard/send_email.coffee

```diff
@@ -15,147 +15,154 @@ create_email_content_table = -> window.InstructorDashboard.util.create_email_con
 create_email_message_views = -> window.InstructorDashboard.util.create_email_message_views.apply this, arguments
```

Comment:
This file retabbed for standardization, [view with `?w=1` for clean diff.](https://github.com/edx/edx-platform/pull/12480/files?w=1#diff-84c026f6b037df9822fa7e5caf4ff813)